src/plugins/discover/kibana.json

```diff
@@ -17,7 +17,7 @@
     "dataViewEditor"
   ],
   "optionalPlugins": ["home", "share", "usageCollection", "spaces", "triggersActionsUi"],
-  "requiredBundles": ["kibanaUtils", "home", "kibanaReact", "dataViews"],
+  "requiredBundles": ["kibanaUtils", "home", "kibanaReact", "dataViews", "sharedUX"],
```

Comment:
Should this be a plugin or a bundle? Is there a difference in the order of execution of lifecycle methods?